src/components/Navigation/Navigation.jsx

```diff
@@ -27,29 +34,42 @@ export const Navigation = () => {
             </Link>
           </li>
         )}
-        
+
         {token && (
           <>
-          <li className="list-inline-item">
-          <div
-            className="avatar avatar-text-xs avatar-text img-round user-profile"
-            role="img"
-            alt="Avatar"
-          >
-            MD
-          </div>
-        </li>
-          <li className="list-inline-item">
-            <button
-              className="btn btn-icon-primary user-signout" onClick={()=>LogoutHandler()}
-            >
-              <Link to="/">
-                <i className="fa fa-sign-out fa-2x"></i>
-              </Link>
-            </button>
-          </li>
+            <li className="list-inline-item">
+              <div className="avatar avatar-xs">
+                <img
+                  className="img-responsive img-round"
+                  src="https://i.postimg.cc/28Zcgq1j/avatar.png"
+                  alt="Avatar"
+                />
+              </div>
+            </li>
+            <li className="list-inline-item">
+              <button
+                className="btn btn-icon-primary user-signout"
+                onClick={() => LogoutHandler()}
```

Comment:
instead of passing the logoutHandler function to the anonymous callback function, just pass it directly like that: onClick={LogoutHandler}

Reply:
Ok, I will check that